### DISCUSSION TITLE
Link is not shown completely

### DISCUSSION THREAD

**Discussion by Paradox-2** [jaykali/maskphish#51] [Q&A] [answered]
Mask phish not working, the link I get is like https://youtube.com/free-premium-user@ . Then the rest of the link is missing. Please help me.

**Comment by jaykali** [comment]
Please share screenshot.

**Comment by ingenium-logic** [comment]
Why ? --Envoyé à partir d’un Smartphone Android avec GMX Mail.Le 07/07/2021, 08:50 JayKali ***@***.***> a écrit:

   
  Please share screenshot.

**Reply by jaykali** [reply]
To solve your problem we need to see the screenshot of the error.

**Reply by Paradox-2** [reply]
Error is nothing but the shorterned link is not shown at the final link.

**Reply by Paradox-2** [reply]
Its like https://google.com-get-free-gems@

**Reply by Paradox-2** [reply]
The rest of the link, which is the shortned is not showing

**Reply by jaykali** [reply]
are you using the latest version of it?

**Reply by Paradox-2** [reply]
I found an way, just creating an fake link by our own and shortning the link and pasting it at last from @ which the google avoids.

**Comment by jaykali** [comment]
Please delete the maskphish directory and git clone it again then try.

**Reply by Paradox-2** [reply]
I tried that way 2 times

**Reply by Paradox-2** [reply]
No improvement, Internet connection is stable.

**Reply by Paradox-2** [reply]
I know the exact logic behind this tool, but I am not an developer to create my own. Please use an better link shortning api to get it done faster.

**Reply by jaykali** [reply]
I don't know what is the problem. If you share the screenshot then we can find a way to improve maskphish. It's grate  to hear that you found a way to do this. You got the right way. This is a very simple but effective tool.

**Reply by Paradox-2** [reply]
I am ready to help you to fix the bugs.

**Reply by Paradox-2** [reply]
![Uploading Screenshot_2021-07-09-17-44-16-061_com.termux.jpg…]()

**Reply by Paradox-2** [reply]
![Screenshot_2021-07-09-17-44-16-061_com termux](https://user-images.githubusercontent.com/86906968/125076594-bab8ce80-e0dd-11eb-9d7a-b36e0f6c81d6.jpg)

**Reply by Paradox-2** [reply]
Here is the ss

**Comment by jaykali** [comment]
Here we did exactly your things and we didn't face any problem.
![maskphish](https://user-images.githubusercontent.com/62602243/125077366-a7f2c980-e0de-11eb-8760-f50a57a271ba.png)

**Comment by jaykali** [comment]
You can simply add you phishing link (without http) after @ 
Like [https://facebook.com-get-blue-badge@a6eb20c7c4da.ngrok.io](https://facebook.com-get-blue-badge@a6eb20c7c4da.ngrok.io)

**Reply by Paradox-2** [reply]
So why getting error for me? 
Can you explain more.
Mine is Android 10.

**Reply by jaykali** [reply]
Still didn't get why is the problem comes for you. This is not depending on your android version.

**Reply by jaykali** [reply]
Are you using tor and vpn services?

**Reply by Paradox-2** [reply]
No, I don't use any VPN.

**Comment by jaykali** [comment, marked as answer]
Your ISP (Internet Service Provider) is blocking is.gd services. Try to oepn [https://is.gd](https://is.gd) website on your browser. If this website opens then Maskphish also work otherwise you need to use VPN to run Maskphish.
